packages/rum/src/viewCollection.ts

```diff
@@ -184,7 +184,9 @@ function newView(
     },
     updateLocation(newLocation: Location) {
       location = { ...newLocation }
+      this.url = location.href
     },
+    url: location.href,
```

Comment:
what about a getter instead of maintaining a new property?
```suggestion
    updateLocation(newLocation: Location) {
      location = { ...newLocation }
    },
    get url() { return location.href },
```